src/ui/RealmBrowser/Content/Table/HeaderCell.tsx

```diff
@@ -41,17 +43,42 @@ const getPropertyType = (property: Realm.ObjectSchemaProperty) => {
   }
 };
 
+const optionalMarkOptOut = ['list', 'dictionary', 'set'];
```

Comment:
```suggestion
const NON_OPTIONAL_TYPES = ['list', 'dictionary', 'set'];
```
Following the pattern used below.